src/notebook/model.ts

```diff
@@ -496,141 +317,153 @@ class NotebookModel implements INotebookModel {
     }
     cells.clear();
     this._cells = null;
```

Comment:
Agreed that it should be disposable.  I think we'll be absorbing `ObservableList` into `jupyter-js-utils` instead of `phopshor`.